### PR TITLE
added anew color of hovered tiles during drag and drop if drop

### PR DESCRIPTION
**Pull request by andreasharbo**
is possible. Also autoformatted tileLayout.py

### CODE REVIEW

**Comment by andreasharbo** [comment]
This only covers drag and drop. Maybe you can think of a solution for resize in a nother PR 💯

File: QTileLayout/tileLayout.py [hunk 1]
```diff
@@ -189,6 +190,10 @@ def setColorDragAndDrop(self, color: tuple):
         """the tile color during drag and drop"""
         self.colorMap['drag_and_drop'] = color
 
+    def setColorEmptyCheck(self, color: tuple):
+        """the tile color during if empty during move"""
```
Comment (arnaudframmery) [inline QTileLayout/tileLayout.py]: Small typo in comment

File: QTileLayout/tileLayout.py [hunk 2]
```diff
@@ -298,12 +303,16 @@ def hardSplitTiles(self, fromRow, fromColumn, tilesToSplit):
 
     def isAreaEmpty(self, fromRow, fromColumn, rowSpan, columnSpan):
         """checks if the given space is free from widgets"""
+        self.changeTilesColor('resize')
```
Comment (arnaudframmery) [inline QTileLayout/tileLayout.py]: It should be the 'drag_and_drop' color I think

**Review by arnaudframmery — CHANGES_REQUESTED** [review]
I love the result !

It would be great if you can add a line in the readme about the method `setColorEmptyCheck()` :)

There is just one thing that can be annoying :/
If someone use the method `isAreaEmpty()` just to check if an area of the board is empty, this area will be colored. I see 2 easy solutions for that:

- You can maybe think about a parameter in the `isAreaEmpty()` to trigger coloration
- Or triggering coloration in `__isDropPossible()` in fonction of the `isAreaEmpty()` result (I think personnaly that it's the best option)

Let me know what you think about it :)

**Comment by andreasharbo** [comment]
I have pushed a fix to your comments :)

I decided to add a parameter to `isAreaEmpty` method, that defaults to False. This way no existing implementations are affected by it. Also this way other functions can utilize the coloring without implementing it themselves. 

Fx if we want to color the tiles that we are resizing to, all we have to do is call `isAreaEmpty` during resizing with the parameter color=True.

Let me know if that makes sense to you!

**Review by arnaudframmery — APPROVED** [review]
If it's your last commit I'm okay to merge it 👍

**Comment by andreasharbo** [comment]
i have no more to add for now :)

**Comment by arnaudframmery** [comment]
Nice ! I will update pypi package by the end of the week

Thank you for the contribution :)

**Comment by andreasharbo** [comment]
Sounds great! :)

**Pull request merged**